src/registrar/assets/sass/_theme/_forms.scss

```diff
@@ -31,3 +31,10 @@
   padding-left: 0;
   border-left: none;
 }
+
+legend.float-left-tablet + button.float-right-tablet {
+  margin-top: .5rem;
+  @include at-media('tablet') {
```

Comment:
**Question:**  `@include at-media('media--lg')` pattern instead?

Reply:
I've never seen this token before. Do you have a link to docs?

In any case, we've used the mobile, tablet and desktop tokens for USWDS media query mixin so we either change all or stick with our convention.

Reply:
Ah my apologies, I smooth brained -- I meant `@include at-media(mobile-lg)`. That's the more common one I've been using, but I do see both `tablet` and `mobile-lg` used on `getgov-home` and `manage.get.gov` so no fuss on what y'all already have!